src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -112,18 +113,26 @@ void Notifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::st
     std::string content_type = (format == XML)? "application/xml" : "application/json";
 
 #ifdef SEND_BLOCKING
-    httpRequestSend(host,
-                    port,
-                    protocol,
-                    "POST",
-                    tenant,
-                    spathList,
-                    xauthToken,
-                    uriPath,
-                    content_type,
-                    payload,
-                    true,
-                    NOTIFICATION_WAIT_MODE);
+    std::string r;
+
+    r = httpRequestSend(host,
+                        port,
+                        protocol,
+                        "POST",
+                        tenant,
+                        spathList,
+                        xauthToken,
+                        uriPath,
+                        content_type,
+                        payload,
+                        true,
+                        NOTIFICATION_WAIT_MODE);
+
+    if ((r != "") && (r != "error"))
```

Comment:
`(r != "") && (r != "error")` means that the notification was send and the other end responsed ok. Is that correct?